app/[locale]/locale-test/page.tsx

```diff
@@ -0,0 +1,31 @@
+'use client'
+
```

Comment:
This is just a temporary page where you can see the localization tests in action